src/components/Social_media.jsx

```diff
@@ -0,0 +1,27 @@
+import { FaFacebookF, FaTwitter, FaCamera } from 'react-icons/fa'
+
+export default function Social_media() {
+  const social_media = [
+    { title: 'Facebook', link: '#', icon: <FaFacebookF className="inline-block"/> },
+    { title: 'Twitter', link: '#', icon:<FaTwitter className="inline-block"/> },
+    { title: 'Shot', link: '#', icon:<FaCamera className="inline-block"/> },
```

Comment:
These social media links don't lead anywhere.  please kindly remove it or give a destination so that all features are fully functional